assets/templates/socialaccount/connections.html

```diff
@@ -0,0 +1,47 @@
+{% extends "account/settings/settings_linked_accounts.html" %}
+
+{% block details %}
+
+    {% if form.accounts %}
+        {% if form.non_field_errors %}
+            {% for error in form.non_field_errors %}
+                <div class="alert alert-danger" role="alert">{{ error }}</div>
+            {% endfor %}
+        {% endif %}
+
+        <p>The following external accounts are linked to this account:</p>
+
+        <form method="post" action="{% url 'socialaccount_connections' %}">
+            {% csrf_token %}
+            {% for base_account in form.accounts %}
+                {% with base_account.get_provider_account as account %}
+                    <div class="">
+                        <label for="id_account_{{ base_account.id }}">
+                            <input id="id_account_{{ base_account.id }}" type="radio" name="account"
+                                   value="{{ base_account.id }}"/>
+                            <span class="socialaccount_provider {{ base_account.provider }} {{ account.get_brand.id }}">{{ account.get_brand.name }}</span>
+                            {{ account }}
+                        </label>
+                    </div>
+                {% endwith %}
+            {% endfor %}
+
+            <div>
+                <button type="submit">{% trans 'Remove' %}</button>
+            </div>
+
+        </form>
+
+    {% else %}
+        <p>{% trans 'You currently have no social network accounts connected to this account.' %}</p>
```

Comment:
I'd prefer external accounts over social accounts. Social implies Facebook and Twitter and that's not what we do or currently aim for here.